Tests/Acceptance/DoctrineAccessTokenManagerTest.php

```diff
@@ -109,25 +160,29 @@ public function testClearExpiredWithRefreshToken(): void
             timecop_return();
         }
 
+        $mapFunction = function (RefreshToken $refreshToken): string {
+            return $refreshToken->getIdentifier();
+        };
+
         $this->assertSame(
-            $testData['output'],
-            $em->getRepository(RefreshToken::class)->findBy(['accessToken' => null], ['identifier' => 'ASC'])
+            array_map($mapFunction, $testData['output']),
+            array_map($mapFunction, $em->getRepository(RefreshToken::class)->findBy(['accessToken' => null], ['identifier' => 'ASC']))
         );
     }
 
     private function buildClearExpiredTestDataWithRefreshToken(Client $client): array
     {
         $validRefreshTokens = [
-            $this->buildRefreshToken('1111', '+1 day', $client),
-            $this->buildRefreshToken('2222', '+1 hour', $client),
-            $this->buildRefreshToken('3333', '+1 second', $client),
-            $this->buildRefreshToken('4444', 'now', $client),
+            $this->buildRefreshToken($client, '1111', '+1 day'),
```

Comment:
@franjo-zadelj-trikoder What is the purpose of the signature change?

Reply:
same as above